### PR TITLE
feat(organization): edit counties

### PR DESCRIPTION
**Pull request by purusott**
When an organization is created, the user can update counties in the edit-organizations page.

This is solved by creating a separate file `countiesSelect.tsx` which contains checkboxes rendered by the `useCountiesSearch()`-hook wrapped in a `<form> </form>`.  

-On form submit, the chosen checkboxes will be stored in firebase connected to its _organizationID_ by updating the organization collection. 
-The `useCountiesSearch()`-hook now takes in _organizationID_ as a (optional) parameter to retrieve counties  selected by a specific organization. This is used to pre-select counties when users want to create a board for their organization (see picture below).

How it looks for now:
image

When the user wants to create a board, the chosen counties will be automatically selected in step 2, as shown below:
image

### CODE REVIEW

**Review by oyvindgrutle — APPROVED** [review]

File: next-tavla/app/(admin)/organizations/[id]/page.tsx [hunk 1]
```diff
@@ -51,6 +51,20 @@ async function EditOrganizationPage({ params }: TProps) {
...
+                    <Paragraph>
+                        Når du søker etter stoppesteder vil du søke i alle
+                        fylker. Her kan du velge hvilke fylker du ønsker å
+                        begrense søket til.
+                    </Paragraph>
+                    <CountiesSelect
+                        oid={organization.id}
+                        countiesList={organization.counties}
+                    />
+                </div>
```
Comment (lindtvedtsebastian) [inline next-tavla/app/(admin)/organizations/[id]/page.tsx:67]: Jeg tenker at Heading og Paragraph kan flyttes inn i CountiesSelect

File: next-tavla/app/(admin)/organizations/components/MemberAdministration/CountiesSelect/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    oid,
+    countiesList,
+}: {
+    oid?: TOrganizationID
+    countiesList?: TCountyID[]
+}) {
+    const { counties } = useCountiesSearch(oid)
+
+    return (
+        <>
```
Comment (lindtvedtsebastian) [inline next-tavla/app/(admin)/organizations/components/MemberAdministration/CountiesSelect/index.tsx:19]: Dette fragmentet er vel ikke nødvendig

File: next-tavla/app/(admin)/organizations/components/MemberAdministration/CountiesSelect/index.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            <form
+                action={async (data: FormData) => {
+                    const counties = data.getAll('county') as string[]
+
+                    if (!oid) return null
+                    setCounties(oid, counties)
+                }}
+            >
+                <div
+                    className={`flexColumn flexWrap ${styles.countiesSelectContainer}`}
```
Comment (lindtvedtsebastian) [inline next-tavla/app/(admin)/organizations/components/MemberAdministration/CountiesSelect/index.tsx:29]: Når vi uansett har en klasse kan vi like gjerne legge inn de flex egenskapene i den klassen

File: next-tavla/app/(admin)/organizations/components/MemberAdministration/styles.module.css [hunk 4]
```diff
@@ -16,3 +16,6 @@
     gap: 0.5em;
     height: 4rem;
 }
+.countiesSelectContainer {
+    height: 12.5rem;
```
Comment (lindtvedtsebastian) [inline next-tavla/app/(admin)/organizations/components/MemberAdministration/styles.module.css:20]: Bruk grid og grid flow istedenfor dette

File: next-tavla/app/(admin)/organizations/components/MemberAdministration/styles.module.css [hunk 5]
```diff
@@ -16,3 +16,6 @@
     gap: 0.5em;
     height: 4rem;
 }
+.countiesSelectContainer {
```
Comment (lindtvedtsebastian) [inline next-tavla/app/(admin)/organizations/components/MemberAdministration/styles.module.css:19]: Denne klassen kan du legge inn i en fil som ligger inni CountiesSelect mappen

**Review by lindtvedtsebastian — CHANGES_REQUESTED** [review]

**Review by lindtvedtsebastian — CHANGES_REQUESTED** [review]

**Review by lindtvedtsebastian — APPROVED** [review]

**Pull request merged**